.github/workflows/cargo.yml

```diff
@@ -40,6 +40,6 @@ jobs:
       run: |
         cargo install cargo-raze --version 0.12.0
         cargo raze
-        # Ignore manual changes in "errno" crate until fixed in cargo-raze.
+        # Ignore manual changes in "errno,cpufeatures and rustix" crate until fixed in cargo-raze.
```

Comment:
Nit: sort alphabetically (to match files) and add space after the comma.

Reply:
Done